src/vm/typehandle.h

```diff
@@ -419,8 +419,8 @@ class TypeHandle
 #endif 
 
     // Unlike AsMethodTable, GetMethodTable will get the method table
-    // of the type, regardless of whether it is an array etc. Note, however
-    // this method table may be shared, and some types (like TypeByRef), have
+    // of the type, regardless of whether it is an array etc. 
+    // Note, however some types (like TypeByRef), have
```

Comment:
I do not think we have a thing in the system called `TypeByRef`. Should it say `ByRef`?